lib/ansible/utils/display.py

```diff
@@ -207,11 +211,11 @@ def locking_wrapper(*args, **kwargs):
     buffer = tio.buffer
 
     # monkeypatching the underlying file-like object isn't great, but likely safer than subclassing
-    buffer.write = _wrap_with_lock(buffer.write, lock)
-    buffer.flush = _wrap_with_lock(buffer.flush, lock)
+    setattr(buffer, "write", _wrap_with_lock(buffer.write, lock))
+    setattr(buffer, "flush", _wrap_with_lock(buffer.flush, lock))
```

Comment:
Workaround for mypy's `method-assign: Cannot assign to a method`.

Reply:
I would just use the normal syntax and add a `type: ignore`. I don't think it makes sense to have less readable code just to satisfy the type checker here.

Reply:
Yes, go for assignment but also don't use a broad ignore. Ignore only one specific violation via `# type: ignore[method-assign]`.

Reply:
Right, I was just too lazy to type the full comment :).